.github/workflows/release_build.yaml

```diff
@@ -7,30 +7,31 @@ on:
     tags:
       - "*.*.*"
   workflow_dispatch:
+  pull_request: 
 
 jobs:
-  docker:
+  build-and-push:
     runs-on: ubuntu-latest
```

Comment:
I don't kow if its a heavy work but is it possible to run onto Alpine instead of Ubuntu to decrease image size ?

Reply:
https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories

Only ubuntu is available

Reply:
For the docker image, alpine is already used. 
See this : https://github.com/tydom2mqtt/tydom2mqtt/blob/master/Dockerfile